src/btree/bt_page.c

```diff
@@ -204,7 +258,7 @@ __page_unpack_deltas_internal(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ITEM *
     WT_DECL_RET;
     size_t *delta_size_each;
     size_t idx;
-    uint32_t d;
+    uint32_t i;
```

Comment:
Why define it to uint32_t and we need to cast in the following code? Can this be the same type as delta_size?

Reply:
Fixed in 7f216e3f25678cf1ad74ac56e2e6cbe04b4d0094